terraform/modules/subnets/variables.tf

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
the formatting of this file looks a little bit broken. Maybe try to run the formatter

Reply:
done